app/src/pages/playground/PlaygroundDatasetExamplesTable.tsx

```diff
@@ -470,11 +469,17 @@ export function PlaygroundDatasetExamplesTable({
   );
 
   const [dialog, setDialog] = useState<ReactNode>(null);
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
```

Comment:
I'm not sure why this state or the useEffect is necessary. Can't you just tell if there's a dialog based on the truthiness of the dialog?